aldryn_faq/urls.py

```diff
@@ -10,6 +10,16 @@
 urlpatterns = patterns('',
     url(r'^$', FaqByCategoryListView.as_view(), name='faq-category-list'),
     url(
+        r'^(?P<category_pk>[0-9]+)-(?P<category_slug>[-\w]+)/$',
```

Comment:
Should we allow the slug portion to begin with a dash or numbers? Perhaps a pattern like:
````
r'^(?P<category_pk>[0-9]+)-(?P<category_slug>[a-z][-\w]*)/$'
````

Would be safer?